src/OpenColorIO/Config.cpp

```diff
@@ -1176,6 +1352,29 @@ void Config::validate() const
     }
 
 
+    ///// VIRTUAL DISPLAY.
+
+    if (getMajorVersion() >= 2)
+    {
+        // Confirm shared view exist and do not conflict with views.
+        for (const auto & sharedView : getImpl()->m_virtualDisplay.m_sharedViews)
+        {
+            // Bypass the <USE_DISPLAY_NAME> validation.
+            getImpl()->validateSharedView("virtual_display",
+                                             getImpl()->m_virtualDisplay.m_views,
```

Comment:
[nitpick] Should these three lines be dedented 3 spaces?

Reply:
Fixed